lib/exercism/connection.rb

```diff
@@ -0,0 +1,15 @@
+module Connection
+  def self.included(base)
+    base.extend(Method)
+    base.send(:include, Method)
+  end
+
+  module Method
+    def connect_to(url)
+      Faraday.new(url: url) do |c|
+        c.use Faraday::Response::Logger
+        c.use Faraday::Adapter::NetHttp
+      end
+    end
+  end
```

Comment:
@kotp @nilbus Is this idiomatic Ruby? It seems unnecessarily complicated to me (but I've not written Ruby professionally for a few years, so I don't know anymore.